src/sentry/tasks/clear_expired_rulesnoozes.py

```diff
@@ -0,0 +1,10 @@
+from django.utils import timezone
+
+from sentry.models import RuleSnooze
+from sentry.tasks.base import instrumented_task
+
+
+@instrumented_task(name="sentry.tasks.clear_expired_rulesnoozes", time_limit=65, soft_time_limit=60)
+def clear_expired_rulesnoozes():
```

Comment:
This seems ... too simple? I feel like I am missing something, but I based this off of the `GroupSnooze` one and the only other thing it does is check ignored and update statuses.

Reply:
I also plan to add a simple delete endpoint for manual unsnoozing once the other PR is merged.